README.md

```diff
@@ -1,28 +1,54 @@
-    3. When you are happy with this script, double click on the xml file again and then click `Publish` in the dialog box that pops up. This will generate the same script again, and then run it on the SQL server, creating your new database in the process.
-4. To populate the database:
-    1. Run the stored procedure `uspPopulateCalendarTable`
-    2. Run the a SQL command based on the script in `source/Scripts/PopulateFeatureFlags.sql`. To make a dev database, you will want to set the three numbers being inserted to `1, 1, 1` instead of the default `0, 0, 0`.
-    3. Run the stored procedure `uspGenerate`
+1. Follow the instructions for setting up the NTBS application in the [ntbs_Beta repository](https://github.com/publichealthengland/ntbs_Beta/blob/master/ntbs-service/README.md).
+1. Follow the instructions for setting up the NTBS migration database in the [ntbs-data-migration repository](https://github.com/publichealthengland/ntbs-data-migration/blob/master/README.md).
+1. Restore a backup of the geography database by carrying out the following steps:
+    i. Connect to the `ntbs-ops-dbs.uksouth.cloudapp.azure.com\NTBS` database server in SSMS.
+    i. In the `Object Explorer` panel, right-click on the `int-geography` database and select `Tasks` -> `Back Up...`.
+    i. Click `OK`.
+    i. After some time, you should see a message informing you that the back up has been successful.
+    i. Connect to the `ntbs-ops-dbs.uksouth.cloudapp.azure.com` VM via Remote Desktop (the username and password can be found in the `ntbs-ops-dbs-credentials` secret in Azure) and locate the back up file you just created.
+    i. Zip the file.
+    i. Copy the file to a temporary location on your development machine.
+    i. Unzip the file.
+    i. In SSMS, connect to your SQL Server instance.
+    i. In the `Object Explorer` panel, right-click on `Databases` and select `Restore Database...`.
+    i. Select the `Device` radio button, and add the relevant back up file via the `...` button.
+    i. Change the name of the database from `int-geography` to `geography`.
+    i. Click `OK`.
+    i. You should see a message saying that the database has restored successfully.
+1. Set up specimen matching database:
+    i. Clone the [specimen-matching repository](https://github.com/publichealthengland/ntbs-specimen-matching).
+    i. Make a copy of the `DEV-specimen-matching.publish.xml` file, named `DEV-USER-specimen-matching.publish.xml`. This file will be ignored by git.
+    i. If the instance of SQL Server that you are using is not at `localhost` then update the `Data Source` in the `TargetConnectionString`.
+    i. Double click on this config in the `Solution Explorer` panel in Visual Studio to publish the codebase. This will build the relevant views and set up the relevant tables.
```

Comment:
Very minor but this markdown doesn't render properly here on the GitHub website. I think it is because all ordered lists must be indexed with Arabic numbers in the markdown file, regardless of nesting level. It will then render the numbers in different numeral sets for the different nesting levels.